src/app/components/UserMenu/index.tsx

```diff
@@ -75,6 +76,14 @@ export default function UserMenu() {
           <GearIcon className="h-5 w-5 mr-2 text-gray-500" />
           Settings
         </Menu.ItemButton>
+        <Menu.ItemButton
+          onClick={() => {
+            window.open("https://feedback.getalby.com");
```

Comment:
you can use `utils.openUrl` here: https://github.com/getAlby/lightning-browser-extension/blob/00c03bf895a295b90813de1b68d73f05959ea4f0/src/common/lib/utils.ts#L90